Add HealthCheckInfo (#366)
submariner-io/submariner-website: src/content/operations/troubleshooting/_index.en.md

#### TBD

-->

### Connectivity Issues

Submariner deployment completed successfully but Services/Pods on one cluster are unable to connect to Services on another cluster. This can

> be due to multiple factors outlined.

Reviewer: ```suggestion
be due to multiple factors outlined below.
```

> #### Check the connection statistics

Reviewer 2: ```suggestion
#### Check the Connection Statistics
```

Author: Done.

> If you are unable to connect to a remote cluster, check the connection statistics.

Reviewer: ```suggestion
If you are unable to connect to a remote cluster, check its connection status in the Gateway resource.
```

> `kubectl describe gateways.submariner.io -n submariner-operator`

Reviewer 3: I think it's suggested to use: `kubectl describe Gateway` instead.

Sample output:

```yaml
    Endpoint:
      Backend:          strongswan
      cable_name:       submariner-cable-cluster1-172-17-0-8
      cluster_id:       cluster1
      Health Check IP:  10.1.1.1
      Hostname:         cluster1-worker
      nat_enabled:      false
      private_ip:       172.17.0.8
      public_ip:
      Subnets:
        100.1.0.0/16
        10.1.0.0/16
    Latency:
      Average RTT:   668235
      Last RTT:      878320
      Max RTT:       3084294
      Min RTT:       165022
      Stddev RTT:    371873
    Status:          connected
    Status Message:  Connected to 172.17.0.8:4500 - encryption alg=AES_GCM_16, keysize=128 rekey-time=12950
```

> The gateway pings the 'Health Check IP' of the endpoint,  the connection status will be marked as an error if it fails to reach the IP.

Reviewer 3: - Remove double space after comma.

> The gateway pings the 'Health Check IP' of the endpoint,  the connection status will be marked as an error if it fails to reach the IP.

Reviewer 2: ```suggestion
The Gateway uses the 'Health Check IP' of the endpoint to verify connectivity. The connection status will be marked as `error` if it fails to reach this IP.
```

> The Gateway Engine uses the `Health Check IP` of the endpoint to verify connectivity. The connection `Status` will be marked as `error`

Reviewer 2: I would add a note here saying that this feature is only available for non-Globalnet deployments at the moment.

Author: Done.

if it cannot reach this IP  and the `Status Message` will provide more information about

> the possible reason for failure. It also provides the statistics for the connection.

Reviewer 2: The statistics, at least in this output, are not clear or actionable (without the units).

Author: Done.

> if it cannot reach this IP  and the `Status Message` will provide more information about
> the possible reason for failure. It also provides the statistics for the connection.

Reviewer 2: ```suggestion
if it cannot reach this IP  and the `Status Message` will provide more information about
the possible failure reason. It also provides the statistics for the connection.
```

> The Gateway uses the 'Health Check IP' of the endpoint to verify connectivity. The connection status will be marked as `error`
> if it fails to reach this IP. If you are facing connectivity issues status message here should give you more information about
> the possible reason for failure. It also gives you the statistics of the connection.

Reviewer: ```suggestion
The Gateway Engine uses the `Health Check IP` of the endpoint to verify connectivity. The connection `Status` will be marked as `error`
if it cannot reach this IP  and the `Status Message` will provide more information about
the possible reason for failure. It also provides the statistics for the connection.
```

#### TBD

-->

### Connectivity Issues

Submariner deployment completed successfully but Services/Pods on one cluster are unable to connect to Services on another cluster. This can
be due to multiple factors outlined below.

#### Check the Connection Statistics

If you are unable to connect to a remote cluster, check its connection status in the Gateway resource.

`kubectl describe Gateway -n submariner-operator`

Sample output:

```yaml
   - endpoint:

>         backend: strongswan

Reviewer 2: ```suggestion
        backend: libreswan
```

Reviewer 2: Let's try to use libreswan in our examples, now that strongswan is being deprecated.

Author: Done.
